lib/solaar/ui/window.py

```diff
@@ -625,22 +608,22 @@ def _update_device_panel(device, panel, buttons, full=False):
 		panel._battery._icon.set_sensitive(False)
 		panel._battery._icon.set_from_icon_name(icon_name, _INFO_ICON_SIZE)
 		panel._battery._text.set_sensitive(True)
-		panel._battery._text.set_markup('<small>%s</small>' % _("unknown"))
+		panel._battery._text.set_markup(_("<small>unknown</small>"))
 	else:
 		charging = device.status.get(_K.BATTERY_CHARGING)
 		icon_name = _icons.battery(battery_level, charging)
 		panel._battery._icon.set_from_icon_name(icon_name, _INFO_ICON_SIZE)
 		panel._battery._icon.set_sensitive(True)
 
 		if isinstance(battery_level, _NamedInt):
-			text = _(str(battery_level))
+			text = _("%(battery_level)s") % { 'battery_level': _(str(battery_level)) }
```

Comment:
battery_level can be NamedInt with value "critical" etc: [/lib/logitech_receiver/hidpp10.py#L102](https://github.com/pwr/Solaar/blob/fa5fba796b87fc1b8f4ce15ac828854ab87a022a/lib/logitech_receiver/hidpp10.py#L102)
Without gettext call here it will not be translated.
Here is translatable strings are defined: [lib/logitech_receiver/i18n.py#L38]( https://github.com/pwr/Solaar/blob/fa5fba796b87fc1b8f4ce15ac828854ab87a022a/lib/logitech_receiver/i18n.py#L38)